.github/workflows/tag-and-publish.yml

```diff
@@ -1,11 +1,7 @@
 name: Publish Package
 
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'Package version to publish'
-        required: true
+  [workflow_dispatch]
 
```

Comment:
@dlozina-macrometa so we won't be bumping the version automatically? How do we change the version? Manually changing setup.py?

Reply:
@koshy-thomas For now, yes, just as we do for JS SDK. But I will need to change how we publish since setup.py is deprecated.